valkey.conf

```diff
@@ -2231,6 +2237,38 @@ notify-keyspace-events ""
 
 ############################### ADVANCED CONFIG ###############################
 
+# It is possible to pin different threads and processes of the server to specific
```

Comment:
Let's put these in the end of the ADVANCED CONFIG section, rather than in the beginning.